### ISSUE TITLE
invalid command 'bdist_wheel' when installing rose-srv6-tutorial dependencies during the update process of the VM

### ISSUE THREAD

**Issue by cscarpitta** [netgroup/rose-vm#20]
Log:

```
#####################################
/home/rose/workspace/rose-vm/scripts/setup_srv6_tutorial.sh


#####################################

-Installing SRv6 Tutorial dependencies

Activating virtual environment

Installing rose-srv6-tutorial dependencies
Collecting git+https://github.com/mininet/mininet.git (from -r requirements.txt (line 4))
  Cloning https://github.com/mininet/mininet.git to /tmp/pip-req-build-ewxoqgrq
  Running command git clone -q https://github.com/mininet/mininet.git /tmp/pip-req-build-ewxoqgrq
Requirement already satisfied (use --upgrade to upgrade): mininet===2.3.0d6 from git+https://github.com/mininet/mininet.git in /home/rose/.envs/mininet-venv/lib/python3.8/site-packages (from -r requirements.txt (line 4))
Requirement already satisfied: python-dotenv==0.13.0 in /home/rose/.envs/mininet-venv/lib/python3.8/site-packages (from -r requirements.txt (line 1)) (0.13.0)
Requirement already satisfied: python-hosts==1.0.0 in /home/rose/.envs/mininet-venv/lib/python3.8/site-packages (from -r requirements.txt (line 2)) (1.0.0)
Requirement already satisfied: win-inet-pton==1.1.0 in /home/rose/.envs/mininet-venv/lib/python3.8/site-packages (from -r requirements.txt (line 3)) (1.1.0)
Requirement already satisfied: setuptools in /home/rose/.envs/mininet-venv/lib/python3.8/site-packages (from mininet===2.3.0d6->-r requirements.txt (line 4)) (44.0.0)
Building wheels for collected packages: mininet
  Building wheel for mininet (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/rose/.envs/mininet-venv/bin/python3 -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-ewxoqgrq/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-ewxoqgrq/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-hjm11geu
       cwd: /tmp/pip-req-build-ewxoqgrq/
  Complete output (8 lines):
  /home/rose/.envs/mininet-venv/lib/python3.8/site-packages/setuptools/dist.py:481: UserWarning: The version specified ('2.3.0d6') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
    warnings.warn(
  usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
     or: setup.py --help [cmd1 cmd2 ...]
     or: setup.py --help-commands
     or: setup.py cmd --help
 
  error: invalid command 'bdist_wheel'
  ----------------------------------------
  ERROR: Failed building wheel for mininet
  Running setup.py clean for mininet
Failed to build mininet

Deactivating virtual environment

Creating starter script for 3routers topology

/home/rose/workspace/rose-srv6-tutorial/nets/3routers/starter.sh already exists. Skipping.

Creating starter script for 8routers topology

/home/rose/workspace/rose-srv6-tutorial/nets/8routers/starter.sh already exists. Skipping.

Creating starter script for 8routers-isis-ipv6 topology

/home/rose/workspace/rose-srv6-tutorial/nets/8routers-isis-ipv6/starter.sh already exists. Skipping.

Creating starter script for 8r-1c-in-band-isis topology

/home/rose/workspace/rose-srv6-tutorial/nets/8r-1c-in-band-isis/starter.sh already exists. Skipping.

Creating starter script for 8r-1c-out-band-isis topology

/home/rose/workspace/rose-srv6-tutorial/nets/8r-1c-out-band-isis/starter.sh already exists. Skipping.

Creating starter script for 8r-1c-srv6-pm topology

/home/rose/workspace/rose-srv6-tutorial/nets/8r-1c-srv6-pm/starter.sh already exists. Skipping.

Done


```

**Comment by cscarpitta** [comment]
The **wheel** package required to install Mininet is missing.
I open a new PR in the [rose-vm-build](https://github.com/netgroup/rose-vm-build) to fix this issue.

**Comment by cscarpitta** [comment]
Fixed in netgroup/rose-vm-build#7
